spec/features/cart/show_spec.rb

```diff
@@ -72,22 +72,22 @@
     end
   end
     describe "i can increment/decrement items" do
-      it "increments (only to inventory limit) and decrements" do
-        @mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
-        @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      it "increments (only to inventory limit) and decrements to zero" do
+        mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+        meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
 
-        @tire = @meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 10)
-        @paper = @mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
-        @pencil = @mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
-        visit "/items/#{@paper.id}"
+        tire = meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 10)
+        paper = mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 3)
+        pencil = mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+        visit "/items/#{paper.id}"
         click_on "Add To Cart"
-        visit "/items/#{@tire.id}"
+        visit "/items/#{tire.id}"
         click_on "Add To Cart"
-        visit "/items/#{@pencil.id}"
+        visit "/items/#{pencil.id}"
         click_on "Add To Cart"
         visit '/cart'
 
-        within "#cart-item-#{@paper.id}" do
+        within "#cart-item-#{paper.id}" do
           expect(page).to have_content("1")
           click_button "Increase"
```

Comment:
For the sake of clarity for the user, should increase change to a "+" and decrease to a "-"?